dot-base/icd-10-api
# feat: return only complete codesystem concepts

# Ticket 🎫

This closes #60.

# Description 📖
This filters the search results for presence of the properties `code` and `display`.

# How to Test? 🧪

Remove a code or a display property from the ICD-10 CodeSystem and search for this code. It should not appear in the search results anymore.

# Is there something controversial ? 🚨

This should log an error, so that the incomplete codes are not dropped silently.

# Checklist ✅
